modules/oc-mirror-oci-format.adoc

```diff
@@ -5,12 +5,25 @@
 
 :_content-type: PROCEDURE
 [id="oc-mirror-oci-format_{context}"]
-= Mirroring file-based catalog Operator images in OCI format
+= Including local OCI Operator catalogs
 
-You can use the oc-mirror plugin to mirror Operators in the Open Container Initiative (OCI) image format, instead of Docker v2 format. You can copy Operator images to a file-based catalog on disk in OCI format. Then you can copy local OCI images to your target mirror registry.
+While mirroring {product-title} releases, Operator catalogs, and additional images from a registry to a partially disconnected cluster, you can include Operator catalog images from a local file-based catalog on disk. The local catalog must be in the Open Container Initiative (OCI) format.
 
-:FeatureName: Using the oc-mirror plugin to mirror Operator images in OCI format
-include::snippets/technology-preview.adoc[]
+The local catalog and its contents are mirrored to your target mirror registry based on the filtering information in the image set configuration file.
+
+[IMPORTANT]
+====
+When mirroring local OCI catalogs, any {product-title} releases or additional images that you want to mirror along with the local OCI-formatted catalog must be pulled from a registry.
+
+You cannot mirror OCI catalogs along with an oc-mirror image set file on disk.
+====
+
+One example use case for using the OCI feature is if you have a CI/CD system building an OCI catalog to a location on disk, and you want to mirror that OCI catalog along with an {product-title} release to your mirror registry.
+
+[NOTE]
+====
+You cannot use the OCI local catalogs feature of the oc-mirror plugin to copy a catalog locally and convert it to OCI format as a first step to mirroring to a fully disconnected cluster.
```

Comment:
Hi @bergerhoffer 
I'm not sure if this confuses users maybe? maybe mention this is only for those who used it in 4.12 to copy?

Reply:
Sure, I can update it to start with the "If you used the TP version..." like I've done elsewhere. Thanks!